src/store/rule/reducer.ts

```diff
@@ -64,13 +64,15 @@ const deletedRule = createReducer<DeleteResponse<string | null>, ActionTypes>(in
     .handleAction(actions.deleteRule.request, () => initialState.deleteRule)
     .handleAction(actions.deleteRule.success, (store, action) => action.payload)
     .handleAction(actions.deleteRule.failure, (store, action) => action.payload)
-    .handleAction(actions.setDeletedRule, (store, action) => action.payload);
+    .handleAction(actions.setDeletedRule, (store, action) => action.payload)
+    .handleAction(actions.resetDeletedRule, () => initialState.deleteRule)
```

Comment:
Codacy has a fix for the issue: [Replace `··.handleAction(actions.resetDeletedRule,·()·=>·initialState.deleteRule)` with `.handleAction(actions.resetDeletedRule,·()·=>·initialState.deleteRule);`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  .handleAction(actions.resetDeletedRule, () => initialState.deleteRule);
```